src/actions/admin.js

```diff
@@ -70,6 +70,16 @@ export function testEmail(config) {
     );
 }
 
+export function testS3Connection(config) {
+    return bindClientFunc(
+        Client4.testS3Connection,
+        AdminTypes.TEST_S3_REQUEST,
+        AdminTypes.TEST_S3_SUCCESS,
+        AdminTypes.TEST_S3_FAILURE,
+        config
+    );
+}
```

Comment:
Oh sorry, can you add a quick unit test for this action?

Reply:
added test, please let me know if that is correct. thanks @jwilander